tests/Makefile

```diff
@@ -1,7 +1,7 @@
-CFLAGS=-Wall -std=c89
+CFLAGS=-Wall -std=c89 -O1
 .PHONY: all clean
 
-all: rollup rollup2 rollupchild rollupchild2
+all: rollup rollup2 rollupchild rollupchild2 pincpu
```

Comment:
Move pincpu to the front so that all the lists are the same.  Possibly, `all` is the wrong idea and pincpu does not belong in this list at all, since the others are needed for the rollup tests.